docs/en/dataset_prepare.md

```diff
@@ -376,3 +377,58 @@ python tools/convert_datasets/isaid.py /path/to/iSAID
 ```
 
 In our default setting (`patch_width`=896, `patch_height`=896,　`overlap_area`=384), it will generate 33978 images for training and 11644 images for validation.
+
+
+### Delving into High-Quality Synthetic Face Occlusion Segmentation Datasets
+
+The dataset is generated by two techniques, Naturalistic occlusion generation, Random occlusion generation. you must install face-occlusion-generation and dataset. see more guide in https://github.com/kennyvoo/face-occlusion-generation.git
+
+  
+## Dataset Preparation
+
+Please download the masks from this [drive](https://drive.google.com/drive/folders/15nZETWlGMdcKY6aHbchRsWkUI42KTNs5?usp=sharing) and the images from [CelebAMask-HQ](https://github.com/switchablenorms/CelebAMask-HQ), [11k Hands](https://sites.google.com/view/11khands) and [DTD](https://www.robots.ox.ac.uk/~vgg/data/dtd/). 
+
+The extracted and upsampled COCO objects images and masks can be found in this [drive](https://drive.google.com/drive/folders/15nZETWlGMdcKY6aHbchRsWkUI42KTNs5?usp=sharing).
+
+Please extract CelebAMask-HQ and 11k Hands images based on the splits found in [drive](https://drive.google.com/drive/folders/15nZETWlGMdcKY6aHbchRsWkUI42KTNs5?usp=sharing). 
+
+**Dataset Organization:**
+
+```none
+
+├── dataset
+│   ├── CelebAMask-HQ-WO-Train_img
+│   │   ├── {image}.jpg
+│   ├── CelebAMask-HQ-WO-Train_mask
+│   │   ├── {mask}.png
+│   ├── DTD
+│   │   ├── images
+│   │   │   ├── {classA}
+│   │   │   │   ├── {image}.jpg
+│   │   │   ├── {classB}
+│   │   │   │   ├── {image}.jpg
+│   ├── 11k-hands_img
+│   │   ├── {image}.jpg
+│   ├── 11k-hands_mask
+│   │   ├── {mask}.png
+│   ├── object_image_sr
+│   │   ├── {image}.jpg
+│   ├── object_image_x4
+│   │   ├── {mask}.png
+
+```
+
+## Data Generation
+
+Example script to generate NatOcc dataset 
+
+bash NatOcc.sh
+
+Example script to generate RandOcc dataset
+
+bash RandOcc.sh
+<!-- #endregion -->
+
+```python
+
+```
```

Comment:
Is it possible to use codes from the original [repo](https://github.com/kennyvoo/face-occlusion-generation) as a reference and redevelop a script?

Reply:
done